package.json

```diff
@@ -5,8 +5,12 @@
   "main": "index.js",
   "scripts": {
     "start": "node src/buildServer.js",
-    "test": "mocha 'test/catRoutes.test.js' --exit",
-    "jest": "jest"
+    "test": "mocha 'test/catRoutes.test.js' --exit",  
+    "jest": "jest --forceExit",
```

Comment:
I'm ok with this for now, it'd be nice to add a comment explaining why this is necessary. and perhaps we can spend a bit of time debugging it later and figuring out whether there's a better solution. I wonder whether it's partially related to my comments on #11

Reply:
It is. No current working alternative although there are gazillion suggested solutions online (that I couldn't get to work!). To be revisited, as I believe that using -forceExit is bad practice.